src/Microsoft.OpenApi/Models/OpenApiParameter.cs

```diff
@@ -101,7 +101,7 @@ public class OpenApiParameter : IOpenApiSerializable, IOpenApiReferenceable, IOp
         /// Furthermore, if referencing a schema which contains an example,
         /// the examples value SHALL override the example provided by the schema.
         /// </summary>
-        public IList<OpenApiExample> Examples { get; set; } = new List<OpenApiExample>();
+        public IDictionary<string,OpenApiExample> Examples { get; set; } = new Dictionary<string,OpenApiExample>();
```

Comment:
Please make sure this change follows up the v3.x spec.

Reply:
Yes, I triple checked that examples is a map, not a list.  https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#fixed-fields-10

Reply:
Thanks！